packages/parse5-htmlparser2-tree-adapter/package.json

```diff
@@ -26,7 +26,7 @@
         "parse5": "^7.0.0"
     },
     "scripts": {
-        "build:cjs": "tsc --module CommonJS --target ES6 --outDir dist/cjs && echo '{\"type\":\"commonjs\"}' > dist/cjs/package.json"
+        "build:cjs": "tsc --moduleResolution node10 --module CommonJS --target ES6 --outDir dist/cjs && echo '{\"type\":\"commonjs\"}' > dist/cjs/package.json"
```

Comment:
> 'node10' (previously called 'node') for Node.js versions older than v10, which only support CommonJS require. You probably won’t need to use node10 in modern code.

a) why is such an old thing needed here,
b) what does it have to do with eslint?

Reply:
Typescript will refuse to build with node16 resolution and commonjs module

Our config was wrong all along but the older tsc just didn't complain

Presumably installing tseslint bumped it in the lock file

So either we do this or we fix cjs builds properly (which means we can't just run a build twice but need a separate config which doesn't have type: module or vice versa)

Reply:
I switched my packages to https://github.com/isaacs/tshy to work around this (and improve the source maps situation).

Reply:
Id rather not do that in this pr, especially given @wooorm raising that we need to split PRs up

So maybe let's move ahead with this since the config was wrong before anyway? And I'll open a separate pr some day to use tshy

Reply:
Sounds good!